README.md

```diff
@@ -239,8 +267,19 @@ wget https://raw.githubusercontent.com/pytorch/pytorch/main/benchmarks/distribut
 
 #### Steps
 
+##### SS Containerized Libraries
+
 An example command for running the benchmark on two nodes with four GPUs
-per node:
+per node using SS libraries built into the container:
+
+```
+$> env MASTER_ADDR=`scontrol show hostnames "$SLURM_JOB_NODELIST" | head -n 1` srun --mpi=pmix_v4 -n 8 --ntasks-per-node=4 --distribution=*:block --cpu-bind=socket --ntasks-per-socket=1 -c 72 singularity run --nv --bind /projects --bind $TMPDIR --bind $HOME --bind `pwd` /path/to/sif/pytorch-ngc-hpc-dev--0050511.sif python /projects/benchmarking/public/examples/ddp/benchmark.py >& benchmark-out-n8-ppn4.txt
```

Comment:
This is very minor, but may be make this "pytorch-ngc-hpc-dev--0050511.sif" to some generic name? but leaving it is just fine too. or just remove that extra "-" in "dev--"